modules/SofaExporter/src/SofaExporter/MeshExporter.cpp

```diff
@@ -817,6 +816,84 @@ bool MeshExporter::writeMeshTetgen()
     return true;
 }
 
+bool MeshExporter::writeMeshObj()
```

Comment:
Isn't it a duplication of : sofa-src\modules\SofaExporter\src\SofaExporter\OBJExporter.cpp
which already use  #include <sofa/simulation/ExportOBJVisitor.h>
which call void VisualModelImpl::exportOBJ  on each visulaModel...
.... ok so the answer to my question is not exactly the same... but... what a mess!

Reply:
Good question, I would say use assimp or:

1. Move the code out from VisualModelImpl::exportOBJ which is the most complete inside a class OBJWritter which export the mesh checking which info are available (normals, uv, etc.)
2. Either remove or update ObjExporter (SOFA component) to use ObjWritter internally
3. Do the same for SofaExporter
4. take a coffee
5. Update sofa/simulation/ExportOBJVisitor.h to create a ObjExporter each time a VisualModel is encounter.